xpano/gui/pano_gui.cc

```diff
@@ -343,6 +343,14 @@ Action PanoGui::ResolveFutures() {
     if (!result.pano) {
       info_message_ = fmt::format("Failed to stitch pano {}", result.pano_id);
       spdlog::info(info_message_);
+      switch(result.export_code) {
```

Comment:
Please move the code either to algorithm/algorithm.h or utils/opencv, so that we don't have to include opencv headers from pano_gui.

e.g. in namespace xpano:utils::cv: 
std::string ToString(cv::Stitcher::Status status);